align/schema/constraint.py

```diff
@@ -659,30 +663,23 @@ class DoNotIdentify(SoftConstraint):
 class SymmetricBlocks(SoftConstraint):
     pairs: List[List[str]]
     direction: Literal['H', 'V']
+    mirror: Optional[bool] = True
 
     @types.validator('pairs', allow_reuse=True)
     def pairs_validator(cls, value):
-        '''
-        X = Align(2, 3, 'h_center')
-        Y = Align(4, 5, 'h_center')
-        Align(1, X, Y, 6, 'center')
-
-        '''
-        instances = get_instances_from_hacked_dataclasses(cls._validator_ctx())
+        # instances = get_instances_from_hacked_dataclasses(cls._validator_ctx())
```

Comment:
Please remove this line altogether as its no longer needed.

Reply:
done